mote/.stylelintrc

```diff
@@ -0,0 +1,359 @@
+plugins:
+    - stylelint-selector-bem-pattern
+    - stylelint-scss
+rules:
+    plugin/selector-bem-pattern:
+        componentName: '[A-Z]+'
+        componentSelectors:
+            initial: '^\\.{componentName}(?:-[a-z]+)?$'
+        utilitySelectors: '^\\.u-[a-z]+$'
+
+    # Color (http://stylelint.io/user-guide/rules/#color)
+    color-hex-case: lower
+    color-hex-length: long
+    color-named: never
+    color-no-invalid-hex: true
+
+    # Font family (http://stylelint.io/user-guide/rules/#font-family)
+    font-family-name-quotes: always-where-required
+
+    # Font weight (http://stylelint.io/user-guide/rules/#font-weight)
+    font-weight-notation: numeric
+
+    # Function (http://stylelint.io/user-guide/rules/#function)
+    function-calc-no-unspaced-operator: true
+    function-comma-newline-after: never-multi-line
+    function-comma-newline-before: never-multi-line
+    function-linear-gradient-no-nonstandard-direction: true
+    function-max-empty-lines: 0
```

Comment:
consider making this a warning. sometimes there are cases where a selector has many properties and it would be valuable for readability to insert empty lines strategically